citrination_client/views/model_template/builders/SimpleModelTemplateBuilder.py

```diff
@@ -0,0 +1,72 @@
+
+class SimpleModelTemplateBuilder(object):
+    """
+    A low dimensional interface for building model templates, based on a basic "ml config" object that specifies
+    descriptor types and basic input/output roles for each column.
+    """
+
+    def __init__(self):
+        self.descriptors = dict()
+
+    def add_real_descriptor(self, column_name, role, upper_bound, lower_bound, units=None):
```

Comment:
only a true monster would put "upper bound" before "lower bound" in an argument list 👺